packages/near-membrane-base/src/membrane.ts

```diff
@@ -1088,6 +1091,17 @@ export function createMembraneMarshall() {
             private makeProxyStatic(shadowTarget: ShadowTarget) {
                 // assert: trapMutations must be true
                 const { foreignTargetPointer } = this;
+                if (DEV_MODE) {
+                    try {
+                        foreignCallableWarn(
```

Comment:
When does this happen? Would a user see thousands of these?

Reply:
This happens only on the first set of a membraned object when the proxy flips to static or live. This built in dev-mode so not the default build. Still TBD on now to have LWC/LWR build near-membrane. For their builds LWC uses 'developer' and LWR uses 'dev' so there's still some normalizing to do.